python/tvm/relay/build_module.py

```diff
@@ -173,11 +173,17 @@ def build(func,
     else:
         tophub_context = autotvm.util.EmptyContext()
 
+    cfg = BuildConfig.current
+
     with tophub_context:
         func = optimize(func)
         # Fuse ops before running code gen
         func = ir_pass.infer_type(func)
-        func = ir_pass.fuse_ops(func)
+
+        if cfg.pass_enabled("OpFusion"):
+            func = ir_pass.fuse_ops(func, opt_level=2)
```

Comment:
should the `opt_level` here be the default `opt_level` instead of 2, though default is 2? Or should it be 1 since level 1 enables fusion?